### PR TITLE
@vedant-jain03 Added Reverse linked list in python 

### PR DESCRIPTION
**Pull request by vedant-jain03**
issue #193 
@raksha009
Completed!!
->2 test cases
->original code
->explained with comments
->successfully compiled and run

### CODE REVIEW

**Review by raksha009 — CHANGES_REQUESTED** [review]
Plz remove extra files and link it to the folder. Rest LGTM!

**Review by vedant-jain03 — COMMENTED** [review]
Deleted other extra files

**Comment by vedant-jain03** [comment]
> Plz remove extra files and link it to the folder. Rest LGTM!

Deleted other extra files

File: Code/Python/Cycle detection in linked list.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+if(createLinkedList.Detectloop()==True):
+    print("Loop detect")
+else:
+    print("Loop not detected")
+    createLinkedList.printLinkedList()
+    
+    
+#time Complexity:O(n)
+#space complexity:O(1)
+
```
Comment (rudrakshi99) [inline Code/Python/Cycle detection in linked list.py:57]: add space and time complexity at the end of test cases.

File: Code/Python/Cycle detection in linked list.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
+#Program to Detect loop in Linked List;
+#Algorithm is Floyd-Warshall Algorithm
```
Comment (rudrakshi99) [inline Code/Python/Cycle detection in linked list.py:2]: Explain the algorithm.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

File: Code/Python/Cycle detection in linked list.py (hunk 1 above)
Comment (vedant-jain03) [reply Code/Python/Cycle detection in linked list.py:64]: I have added space and time comp. at bottom

File: Code/Python/Cycle detection in linked list.py (hunk 2 above)
Comment (vedant-jain03) [reply Code/Python/Cycle detection in linked list.py:2]: Done 👍

File: Code/Python/Cycle detection in linked list.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+# test case 2:
+# createLinkedList.push(2)
+# createLinkedList.push(3)
+# createLinkedList.push(5)
+# createLinkedList.push(1)
+
+# createLinkedList.head.next.next.next=createLinkedList.head
+# output:
+#     Loop detect
```
Comment (raksha009) [inline Code/Python/Cycle detection in linked list.py]: add space and time complexity here after line 78. And add a new line too.

**Review by raksha009 — CHANGES_REQUESTED** [review]

**Review by raksha009 — CHANGES_REQUESTED** [review]
I think cycle detection is an extra file here. Remove that. Also, link it with respective folder.

**Comment by vedant-jain03** [comment]
> I think cycle detection is an extra file here. Remove that. Also, link it with respective folder.

Done👍

**Comment by raksha009** [comment]
> I think cycle detection is an extra file here. Remove that. Also, link it with respective folder.

link it with respective folder. :)

**Pull request closed without merging**